Sources/CoreAPI/WordPressComRestApi.swift

```diff
@@ -105,6 +105,8 @@ open class WordPressComRestApi: NSObject {
 
     private var useEphemeralSession: Bool
 
+    private var isInvalidated = false
```

Comment:
What do you think about changing those URLSession properties to Optional instead and assigning them to nil in the `invalidateAndCancelTasks` function?